### ISSUE TITLE
Running into "circuit_breaking_exception" when using ScriptFilter

### ISSUE THREAD

**Issue by sethmlarson** [elastic/eland#186]
I run into this issue running many queries requiring a `ScriptFilter`.
Can we get away with caching scripts on the Elasticsearch side or something else we can do to alleviate this?

**Comment by sethmlarson** [comment]
Could this be as simple as changing `inline` to `source` within `ScriptFilter`?

**Comment by mesejo** [comment]
You mean something like:
```
        script = {"source": inline}
        if lang is not None:
            script["lang"] = lang
```
?

**Comment by sethmlarson** [comment]
Yeah that would fix the issue I think? Check out #190 as well.

**Comment by sethmlarson** [comment]
Closed via https://github.com/elastic/eland/pull/191
